core/domain/src/main/java/org/mobicents/ussdgateway/UssdPropertiesManagement.java

```diff
@@ -278,6 +280,15 @@ public void setCdrLoggingTo(CdrLoggedType cdrLoggingTo) {
         this.store();
     }
 
+    public boolean getCdrNavigationSave() {
+        return cdrNavigationSave;
+    }
+
+    public void getCdrNavigationSave(boolean cdrNavigationSave) {
```

Comment:
@abdulazizali77 I believe the method should be called setCdrNavigationSave?

Reply:
@abhayani Thanks Amit, i will address all the review comments in a followup PR